readme.md

```diff
@@ -151,15 +178,15 @@ To use a custom store:
 ```ts
 import rateLimit, { MemoryStore } from 'express-rate-limit'
 
-const apiLimiter = rateLimit({
+const rateLimiter = rateLimit({
 	windowMs: 15 * 60 * 1000, // 15 minutes
 	max: 100, // Limit each IP to 100 requests per `window` (here, per 15 minutes)
 	standardHeaders: true, // Return rate limit info in the `RateLimit-*` headers
 	store: new MemoryStore(),
```

Comment:
maybe we should use the redis store here

Reply:
Done